### ISSUE TITLE
Color issue

### ISSUE THREAD

**Issue by GowriIDPal** [msayan/bread-crumb-view#1]
Hi, I can't able to change the crumb color dynamically when I will import this library as a dependency. Could you please provide any guidance for that?

**Comment by msayan** [comment]
Hi @GowriIDPal ,
Now the library doesn't have any functionality like it. Can you explain more what you want to achieve right now? Do you want every breadcrumb has different colors?

**Comment by GowriIDPal** [comment]
Yes @msayan, I need to change the color dynamically for breadcrumb words. Is it possible?

**Comment by msayan** [comment]
Hello @GowriIDPal ,

Right now it doesn't have that function. I'll be add that feature before weekend.
